### DISCUSSION TITLE
Does $theme_info work?

### DISCUSSION THREAD

**Discussion by steve6375** [a1ive/grub2-filemanager#320] [General]
![image](https://user-images.githubusercontent.com/1469350/103454905-3fb2cd80-4ce0-11eb-898b-f3c7939d71f5.png)

I set theme_info in config
but when I press F1, it is not using it?

set mytheme=slack

```
	export theme_std="(${user})/boot/grubfm/themes/${mytheme}/theme.txt";
	export theme_fm="(${user})/boot/grubfm/themes/${mytheme}/fm.txt";
	export theme_info="(${user})/boot/grubfm/themes/${mytheme}/info.txt";
	export theme=$theme_std
```

**Comment by steve6375** [comment]
My mistake!  theme_info is for the  info menu that you get when you select an ISO or other file  'File Info'
![image](https://user-images.githubusercontent.com/1469350/103455262-940b7c80-4ce3-11eb-9598-5998ca603f57.png)
if  theme_info is not defined or doesnt exist then theme_fm is used.

**Reply by a1ive** [reply]
yes.

**Comment by steve6375** [comment]
OK - one slight problem is that if the user does not use a theme, then F1 does not really work.
If no $theme is defined would it be better not to load the F1 menu when F1 is pressed - just do nothing?

```
	unset theme_std; unset theme_fm;
	unset theme_info;unset theme_hw_grub;unset theme_hw_cpu;unset theme_hw_board;unset theme_hw_ram
	#Note F1 does not work unless a theme is used.
	#Don't use a theme for this menu
	unset theme
	#set resolution (1024x768 recommended)
	set gfxmode=1024x768
	terminal_output gfxterm
	set gfxpayload=1024x768
	#Menu colours - format: text/background  (black background=transparent)
	#Choice: black,blue,brown,cyan,dark-gray,green,light-cyan,light-blue,light-green,light-gray,light-magenta,light-red,magenta,red,white,yellow
	set menu_color_highlight=yellow/black
	set menu_color_normal=light-green/black
	set color_normal=light-gray/black
	#add either your background image or a fill colour - use stretch or normal - .png, .tga, .jpg, or .jpeg
	background_image -m stretch (${user})/boot/grubfm/seven.jpg
```

F1 icon would still appear in menu though...

**Comment by a1ive** [comment]
Now it shows hotkey help menu.
![深度截图_选择区域_20210103120526](https://user-images.githubusercontent.com/10670106/103471480-f8017400-4dbb-11eb-8390-77f708078dd4.png)
https://github.com/a1ive/grub2-filemanager/commit/55e9de569f9768310484d231e8065becb30d67af

**Reply by steve6375** [reply]
thanks
